js/src/mpl_widget.js

```diff
@@ -78,6 +80,11 @@ var MPLCanvasView = widgets.DOMWidgetView.extend({
         this.send_message('initialized');
     },
 
+    update_header_visible: function() {
+        this.header.style.display = this.model.get('header_visible') ? '': 'none';
```

Comment:
You might want to do this when creating `this.header` as well :) In case `header_visible` is False when creating the canvas.

Reply:
Conda was angry at me and it took some effort to be up and running again.

I can add the call to the _init_header method, but the request_resize() causes a libpng error if header_visible is set to False before rendering (#127). 
I tried adding,
```
this.header.style.display = this.model.get('header_visible') ? '': 'none';
```
to _init_header(), while leaving the update method as it is, but this does not change anything. 
I find it a bit strange as the events seem to be set after _init_header(), but unfortunately my javascript knowledge is limited.
The same problem exists for hiding the toolbar, so solving this one would improve that option as well. If you have any suggestions?

Reply:
I would say let's merge it as it is for now. It looks good to me, we'll see later if my comment is needed.